kubernetes_platform/python/kfp/kubernetes/__init__.py

```diff
@@ -23,15 +23,17 @@
     'add_node_selector',
     'add_pod_label',
     'add_pod_annotation',
-    'set_image_pull_secrets'
+    'set_image_pull_secrets',
+    'add_tolerations',
```

Comment:
Can you add that in the right location to respect the order for future modifications?

Reply:
Mind elaborating? The sequence just mirrors the order in which the function was added (which is true of the ones above it as well).

Reply:
```
 'add_node_selector',
    'add_pod_label',
    'add_pod_annotation',
    'add_tolerations',
    'set_image_pull_secrets'
    'set_image_pull_secrets',
```

Reply:
Fixed 👍 .